Intel_SighPP_CMake/Intel_SighPP_CMake/src/Intel_Sight_++.cpp

```diff
@@ -54,6 +61,7 @@ int main(int argc, char** argv)
 	/// 8) -yolo yolo		: This is used to import the darknet-based network (YoloV3).
 	/// 9) -outdoors		: This is used to set up object detection networks, frame resolution and the prioritiser for outdoors environment.
 	/// 10) -indoors		: This is used to set up object detection networks, frame resolution and the prioritiser for indoors environment.
+	/// 11) -covid		: This is used to set up object detection networks, frame resolution and the prioritiser for Outdoors environment and uses a prioritiser that enforces social distancing.
```

Comment:
If the plan is to add more of these, wouldn't it be more appropriate to have it be like: 
```
-env (or some other better word) outdoors
-env indoors
-env covid
```

Reply:
And currently, there can only be one prioritise loaded right? 
So this could make it easy to check if more than one -env (or another better word 😄) flag is used.